db-prod.json

```diff
@@ -0,0 +1,694 @@
+[
```

Comment:
I was gonna say let's not add this, but to make life extra easy let's just merge this in and we can delete later/ add the data to the other seeded data